README.md

```diff
@@ -1,20 +1,149 @@
 <!-- Logo -->
 <p align="center">
-  <img height="128" width="128" src="https://github.com/cerner/terra-core/raw/master/terra.png">
+  <img height="128" width="128" src="https://github.com/cerner/terra-toolkit/raw/master/terra.png">
 </p>
 
 <!-- Name -->
 <h1 align="center">
-  Terra Core
+  Terra Toolkit
 </h1>
 
-[![Build Status](https://travis-ci.org/cerner/terra-core.svg?branch=master)](https://travis-ci.org/cerner/terra-core)
+[![NPM version](http://img.shields.io/npm/v/terra-toolkit.svg)](https://www.npmjs.org/package/terra-toolkit)
+[![Build Status](https://travis-ci.org/cerner/terra-toolkit.svg?branch=master)](https://travis-ci.org/cerner/terra-toolkit)
 
 Terra Toolkit is a utility module used to facilitate development of Terra projects. This toolkit is designed to help support nightwatch testing. It provides configuration, as well as helper functions that can be used during test runs of npm packages within [terra-core][@terra-core] and [terra-clinical][@terra-clinical].
```

Comment:
This seems to imply that it can only be used with terra-core and terra-clinical.  This could actually be used independently in any npm project.  Maybe call out those two projects as examples rather than saying what you have here.